samples/CommunityToolkit.Maui.Sample/Models/AvatarModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace CommunityToolkit.Maui.Sample.Models;
+
+public class AvatarModel
+{
+	public Color BackgroundColor { get; init; } = Colors.Black;
```

Comment:
Can we make it dynamic depends on AppTheme?

Reply:
Let's keep using `Colors.Black` because it works nicely with the default values for `BorderColor` (`Colors.White`) and `TextColor` (`Colors.White`).